driver-sync/src/test/unit/com/mongodb/client/internal/CryptConnectionSpecification.groovy

```diff
@@ -85,7 +82,7 @@ class CryptConnectionSpecification extends Specification {
         then:
         _ * wrappedConnection.getDescription() >> {
             new ConnectionDescription(new ConnectionId(new ServerId(new ClusterId(), new ServerAddress())), 8, STANDALONE,
-                    maxBatchCount, maxDocumentSize, maxMessageSize, [])
+                    1000, 1024 * 16_000, 1024 * 48_000, [])
```

Comment:
Why did maxBatchCount in the expected connection description changed from 4 to 1000?

Reply:
This should be emulating what MongoDB would actually send us, which is 1000

Reply:
I'm not sure why I had it set to 4 previously.  Ideally though we would test that whatever value is advertised here is what's used for the splitting logic.  I can add another test for that.

Reply:
That seems reasonable. Also per slack discussion, add a test of a doc just over 2mb and assert that it throws.

Reply:
Done